src/modules/search/components/menu/types/geoResource/GeoResourceResultsPanel.js

```diff
@@ -0,0 +1,109 @@
+import { html } from 'lit-html';
+import { classMap } from 'lit-html/directives/class-map.js';
+import { $injector } from '../../../../../../injection';
+import { debounced } from '../../../../../../utils/timer';
+import { BaElement } from '../../../../../BaElement';
+import { requestData } from '../resultPanelUtils';
+import css from './geoResourceResultsPanel.css';
+
+
+/**
+ * Displays geoResource search results.
+ * @class
+ * @author taulinger
+ */
+export class GeoResouceResultsPanel extends BaElement {
```

Comment:
can GeoResourceResultsPanel benefit from the css-definitions of AbstractContentPanel?